src/main.rs

```diff
@@ -1,3 +1,5 @@
+#![windows_subsystem = "windows"]
```

Comment:
What do you think about this, instead?

```suggestion
#![cfg_attr(not(debug_assertions), windows_subsystem = "windows")]
```

Reply:
It would also be valuable to have a comment here to explain what this does.